build.sbt

```diff
@@ -25,9 +25,13 @@ scalacOptions := {
 )
 
 libraryDependencies := Seq(
+  "org.joda" % "joda-convert" % "1.7",
```

Comment:
http://www.joda.org/joda-time/upgradeto230.html: 
> Joda-Time uses annotations from Joda-Convert.
In the Java programming language, this dependency is optional, however in Scala it is not.
Scala users must manually add the Joda-Convert v1.2 dependency.